Docs: Fix typo in Storyblok provider (#959)
nuxt/image: docs/content/3.providers/storyblok.md

the logic is:

> - If you are not defining the width or height it's taking the original image size.
> - If you defined only the width or height it will resize proportionaly based on the one defined

Reviewer: ```suggestion
- If you do not define either width or height, the image will not be resized.
- If you define only width or only height the image will be proportionately resized based on the one you defined.
```
